frontend/libs/studio-components-legacy/src/components/StudioCodelistEditor/StudioCodeListEditorRow/StudioCodeListEditorRow.tsx

```
@@ -218,12 +236,15 @@ type TextResourceIdCellProps = {
   currentId: string;
   label: string;
   number: number;
-  onBlurTextResource: (newTextResource: TextResource) => void;
+  onBlurTextResource: (textResource: TextResource) => void;
   onChangeCurrentId: (newId: string) => void;
-  onChangeTextResource: (newTextResource: TextResource) => void;
+  onChangeTextResource: (textResource: TextResource) => void;
   property: CodeListItemTextProperty;
   required: boolean;
   textResources: TextResource[];
+  onCreateTextResource?: ({}: CreateTextResourceInternalArgs) => void;
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)